### ISSUE TITLE
feature request: hide scrollbar

### ISSUE THREAD

**Issue by notslang** [atom-minimap/minimap#75]
This is a feature that I always wanted from the sublime minimap: the ability to hide the editor's scrollbar when the minimap is open. Since the minimap is basically a glorified scrollbar, having the "traditional" scrollbar directly on the side of it seems like a waste of 15x1000px^2 of good screen space.

And as an added bonus, it would look wayyy prettier without the scrollbar:
![image](https://cloud.githubusercontent.com/assets/1049204/2939819/9f85f98a-d95b-11e3-8941-616269aafa75.png)

**Comment by stianhoiland** [comment]
Doesn't work anymore?

**Comment by fundon** [comment]
@stianhoiland https://atom-minimap.github.io/minimap/docs/Quick-Personal-Hacks.md.html

**Comment by stianhoiland** [comment]
Thanks, that worked!

**Comment by ncarreiro** [comment]
For someone who is looking for an updated solution (the CSS Style here is outdated), I found the new atom-minimap Atom documentation here: https://atom-minimap.github.io/minimap/#hiding-scrollbars

Solution:
`atom-text-editor .vertical-scrollbar,
atom-text-editor::shadow .vertical-scrollbar {
  opacity: 0;
  width: 0;
}`

**Comment by colinharris05** [comment]
@ncarreiro that worked perfectly for me, thanks! After finding this I realized I'd rather not have _any_ scrollbars in Atom (e.g. in the files pane as well), but still want them to remain functional with the scroll wheel. 

For anyone who is interested, I found this solution to remove _all_ scrollbars including the one next to the minimap.

Source: https://coderwall.com/p/h_zpfa/hide-scrollbars-in-atom
Solution (added to stylesheet): 
`::-webkit-scrollbar {
    display: none; 
}`
